tests/serializers/test_model.py

```diff
@@ -991,22 +992,22 @@ def test_extra():
     class MyModel:
         # this is not required, but it avoids `__pydantic_fields_set__` being included in `__dict__`
         __slots__ = '__dict__', '__pydantic_fields_set__', '__pydantic_extra__', '__pydantic_private__'
-        field_a: str
         field_b: int
+        field_a: str
 
     schema = core_schema.model_schema(
         MyModel,
         core_schema.model_fields_schema(
             {
-                'field_a': core_schema.model_field(core_schema.bytes_schema()),
                 'field_b': core_schema.model_field(core_schema.int_schema()),
+                'field_a': core_schema.model_field(core_schema.bytes_schema()),
```

Comment:
Why did these have to change? Is this to make the tests change? I would prefer a new self-contained test just for this bit.

Reply:
seperated this.